mypy/main.py

```diff
@@ -205,6 +207,39 @@ def invert_flag_name(flag: str) -> str:
     return '--no-{}'.format(flag[2:])
 
 
+if sys.platform == 'win32':
+    python_executable_prefix = 'py -{}'
+else:
+    python_executable_prefix = 'python{}'
+
+
+def _python_version_from_executable(python_executable: str) -> Tuple[int, int]:
+    try:
+        check = subprocess.check_output([python_executable, '-c',
+                                         'import sys; print(repr(sys.version_info[:2]))'],
+                                        stderr=subprocess.STDOUT).decode()
+    except subprocess.CalledProcessError:
```

Comment:
Can this happen? If the python executable is not found, it throws IOError anyway

Reply:
It seems to be throwing a `FileNotFoundError` in CI, but yes, I should be catching that, not `CalledProcessError`.

Reply:
Ah, I'm thinking in python 2 land. That's still a subclass of IOError, but may as well catch the explicit subclass.

Reply:
> Can `CalledProcessError` happen?

Yes, if for some reason `python27` on my path points to something that is not actually a python interpreter.

I'd say catch both, but in both cases just raise a more useful exception. _"In the face of ambiguity, refuse the temptation to guess"_ and all that.